util/reggen/reg_top.sv.tpl

```diff
@@ -563,13 +563,15 @@ ${reg_hdr}
         flds_no_we |= (not f.swaccess.needs_we()) << f_idx
       if flds_no_we != 0:
         flds_we_masked = f"&({sr_name}_flds_we | {len(sr.fields)}'h{flds_no_we:x})"
+        unused_flds_we_masked = f"^({sr_name}_flds_we & {len(sr.fields)}'h{flds_no_we:x})"
```

Comment:
Oops, I guess this operation didn't quite work for the linter. I'll try changing it to a unary OR of the masked fields.

Reply:
...and I accidentally did declaration and assignment in the same statement, which had a var. Double oops.